config/application.rb

```diff
@@ -21,20 +21,6 @@
 # you've limited to :test, :development, or :production.
 Bundler.require(*Rails.groups)
 
-Raven.configure do |config|
-  config.silence_ready = true
-  config.dsn = ENV["SENTRY_DSN"]
```

Comment:
Same question [as here](https://github.com/alphagov/govuk-attribute-service-prototype/pull/151#discussion_r604092080) about the sentry DSN

Reply:
resolved, see @barrucadu 's answer on other PR